tutorials/tour/unified-types-diagram.dot

```diff
@@ -0,0 +1,13 @@
+digraph UnifiedTypes {
```

Comment:
I agree that the graph is misleading; some ellipses or other indication of what's missing would be helpful.